be/test/olap/rowset/segment_v2/bitmap_index_test.cpp

```diff
@@ -35,6 +35,9 @@
 #include "util/file_utils.h"
 
 namespace doris {
+
+using FileSystemPtr = std::shared_ptr<io::FileSystem>;
```

Comment:
if it is a shared ptr, then name it to  FileSystemSPtr
if it is a unique ptr, then name it to FileSystemUPtr
then we could distinguish them.